.github/workflows/rust.yml

```diff
@@ -173,21 +172,20 @@ jobs:
         uses: ./.github/actions/install-rust-toolchain
         with:
           toolchain: ${{ matrix.toolchain }}
-
-      - name: Install tools
-        uses: taiki-e/install-action@v2
-        with:
-          tool: cargo-hack@0.5.26,cargo-nextest@0.9.37
-
-      - name: Install cargo-make
-        run: cargo install --version 0.36.3 cargo-make
+          working-directory: ${{ matrix.directory }}
+          components: rust-src
 
       - name: Cache Rust dependencies
         uses: Swatinem/rust-cache@v2
         with:
           workspaces: ${{ matrix.directory }}
           key: ${{ matrix.profile }}
 
+      - name: Install tools
+        uses: taiki-e/install-action@v2
+        with:
+          tool: just@1.13.0,cargo-hack@0.5.26,cargo-nextest@0.9.37
```

Comment:
Is there an easy way to centralize where the versions of tools are specified?

Reply:
Not as I know of, no, unless we make everything a single action, which I would avoid. Even then we still have to specify the version in `.justfile`.